boards/arm/mimxrt595_evk/doc/index.rst

```diff
@@ -103,6 +103,9 @@ already supported, which can also be re-used on this mimxrt595_evk board:
 | PM        | on-chip    | power management; uses SoC sleep,   |
 |           |            | deep sleep and deep-powerdown modes |
 +-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | LCDIF; MIPI-DSI. Tested with RM68200|
```

Comment:
Thank you for updating this doc page.  Can we also add to this doc NXP's module part number RK055HDMIPI4MA0 for this display?  And link to this product page https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/5-5-lcd-panel:RK055HDMIPI4MA0.  That way users can easily find this display.  The update does not need to be in this table, but somewhere in this doc.

Reply:
The display that we currently support (using the RM68200 controller) is the [RK055HDMIPI4M](https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/5-5-lcd-panel:RK055HDMIPI4M). It uses a different controller than the updated RK055HDMIPI4MA0, as far as I can tell. I'll add a link to that display here